### ISSUE TITLE
Trouble installing latest version behind proxy

### ISSUE THREAD

**Issue by mrpudn** [giggio/node-chromedriver#78]
I'm trying to install `chromedriver` behind a corporate proxy, and it looks like it's failing to honor proxy settings in the latest version (`2.25.2`):
```
> chromedriver@2.25.2 install .../node_modules/chromedriver
> node install.js

Downloading https://chromedriver.storage.googleapis.com/2.25/chromedriver_mac64.zip
Saving to 
.../chromedriver/chromedriver_mac64.zip
Receiving...
ChromeDriver installation failed Error with http request: { 'cache-control': 'no-cache',
  pragma: 'no-cache',
  'content-type': 'text/html; charset=utf-8',
  'proxy-connection': 'close',
  connection: 'close',
  'content-length': '8129' }
```

As a workaround, we've pinned the version to `2.21.2` (our last known working state), but we'd love to get this resolved so that we can upgrade to the latest version :)

**Comment by giggio** [comment]
On `2.25.2` I changed the default to https, and it seems we are still using the http proxy instead of the https proxy. I am fixing this right now. See comment by @davidblackburn on https://github.com/giggio/node-chromedriver/issues/77#issuecomment-265682078.

**Comment by giggio** [comment]
Published to npm as `2.25.3`.
